airport/airport-web-server/src/main/java/berlin/webapp/config/tablecodes/conditions/ConditionRatingWebUiConfig.java

```diff
@@ -97,7 +97,7 @@ private ConditionRatingWebUiConfig(final Injector injector, final IWebUiBuilder
         final String layout = LayoutComposer.mkGridForMasterFitWidth(1, 2);
 
         final IMaster<ConditionRating> masterConfig = new SimpleMasterBuilder<ConditionRating>().forEntity(ConditionRating.class)
-                .addProp("name").asSinglelineText().also()
+                .addProp("condition").asSinglelineText().also()
```

Comment:
Property condition is now of type `Integer`. A more appropriate editor would be required for both `ConditionRating` master and selection criteria for `ConditionRating` centre.

Reply:
hey @01es i created another PR wherein i resolve this issue as best as i can, and tagged you for a review.